apps/api/src/app/content-templates/usecases/compile-template/compile-template.usecase.ts

```diff
@@ -8,6 +8,21 @@ Handlebars.registerHelper('equals', function (arg1, arg2, options) {
   return arg1 == arg2 ? options.fn(this) : options.inverse(this);
 });
 
+Handlebars.registerHelper('titlecase', function (string) {
+  return string
```

Comment:
```suggestion
  return value?
```

To make sure we are not failing on null or falsy values here.